e2e/playwright/point-click.spec.ts

```diff
@@ -2351,6 +2353,50 @@ chamfer04 = chamfer(extrude001, length = 5, tags = [getOppositeEdge(seg02)])
         })
         await scene.expectPixelColor([146, 146, 146], testPoint, 15)
       })
+
+      // TODO: this test hits a case that happens sometimes in manual testing where
+      // we can't edit the shell right after its creation. As if the codemod didn't fully
+      // register during the initall add flow.
```

Comment:
If anybody's game to help debug this. Otherwise I don't think it's a major blocker to merge but 100% a fairly high priority todo for a follow up

Reply:
I can help soon enough if no one else does

Reply:
Well this is now working so I'm happy! Thanks @lf94

Reply:
https://github.com/KittyCAD/modeling-app/pull/5525#discussion_r1991946566